clients/src/main/java/org/apache/kafka/clients/consumer/internals/MembershipManagerImpl.java

```diff
@@ -568,7 +570,11 @@ public void transitionToFenced() {
     public void transitionToFatal() {
         MemberState previousState = state;
         transitionTo(MemberState.FATAL);
-        log.error("Member {} with epoch {} transitioned to {} state", memberId, memberEpoch, MemberState.FATAL);
+        if (memberId.isEmpty()) {
```

Comment:
Sorry for late review.

According to the log message, it seems this condition should be `!memberId.isEmpty()`, right?

Reply:
Totally! good catch. I can open a minor PR to fix it right away (unless you have it already in-flight?). @lucasbru is not available this week.

Reply:
> I can open a minor PR to fix it right away (unless you have it already in-flight?)

Please take over it if you have bandwidth :)

Reply:
Here it is https://github.com/apache/kafka/pull/16720, thanks!